google/cloud/aiplatform/datasets/tabular_datasets.py

```diff
@@ -0,0 +1,135 @@
+
+    @classmethod
+    def create(
+        cls,
+        display_name: str,
+        gcs_source: Optional[Sequence[str]] = None,
+        bq_source: Optional[str] = None,
+        labels: Optional[Dict] = None,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ) -> "TabularDataset":
+        """Creates a new tabular dataset.
+
+        Args:
+            display_name (str):
+                Required. The user-defined name of the Dataset.
+                The name can be up to 128 characters long and can be consist
+                of any UTF-8 characters.
+            gcs_source: Optional[Sequence[str]]=None:
+                Google Cloud Storage URI(-s) to the
+                input file(s). May contain wildcards. For more
+                information on wildcards, see
+                https://cloud.google.com/storage/docs/gsutil/addlhelp/WildcardNames.
+            bq_source: Optional[str]=None:
+                BigQuery URI to the input table.
+            labels: (Optional[Dict]) = None
+                The labels with user-defined metadata to organize your
+                Datasets.
+
+                Label keys and values can be no longer than 64 characters
+                (Unicode codepoints), can only contain lowercase letters,
+                numeric characters, underscores and dashes. International
+                characters are allowed. No more than 64 user labels can be
+                associated with one Dataset (System labels are excluded).
+
+                See https://goo.gl/xmQnxf for more information and examples
+                of labels. System reserved label keys are prefixed with
+                "aiplatform.googleapis.com/" and are immutable.
+            project: Optional[str]=None,
+                Project to upload this model to. Overrides project set in
+                aiplatform.init.
+            location: Optional[str]=None,
+                Location to upload this model to. Overrides location set in
+                aiplatform.init.
+            credentials: Optional[auth_credentials.Credentials]=None,
+                Custom credentials to use to upload this model. Overrides
+                credentials set in aiplatform.init.
+        Returns:
+            tabular_dataset (TabularDataset):
+                Instantiated representation of the managed tabular dataset resource.
+        """
+
+        metadata_schema_uri = schema.dataset.metadata.tabular
+
+        dataset_obj = super().create(
+            display_name=display_name,
+            metadata_schema_uri=metadata_schema_uri,
+            gcs_source=gcs_source,
+            bq_source=bq_source,
+            labels=labels,
+            project=project,
+            location=location,
+            credentials=credentials,
+        )
+
+        return dataset_obj
+
+    @property
```

Comment:
Don't think the property decorator is relevant here. Property is used to create getters and setters, `import_data` is more of a one-off function.

reference: https://www.programiz.com/python-programming/property